### PR TITLE
LCH-4833: Created group option at application level

### PR DESCRIPTION
**Pull request by ankitsingh0188**

### CODE REVIEW

File: src/PlatformInterface.php [hunk 1]
```diff
@@ -15,6 +15,11 @@ interface PlatformInterface extends CommandQuestionInterface {
 
   public const PLATFORM_ALIAS_KEY = 'platform.alias';
 
+  public const GROUP_CONFIG_LOCATION = [
+    '.commonconsole',
+    'groups',
+  ];
```
Comment (saxumVermes) [inline src/PlatformInterface.php]: Why do we have to specify the same set of constants in two separate interfaces? Can we just make a common one that extends the two? Or any other way? Is it possible one of them will change? In that case it still can be overwritten if they are inherited.

Comment (ankitsingh0188) [reply src/PlatformInterface.php]: Done.

Comment (ankitsingh0188) [reply src/PlatformInterface.php]: Actually, we are using that constants inside the trait only, so created the array in that particular method only.

**Review by jose-oliveira — APPROVED** [review]

**Review by amangrover90 — APPROVED** [review]
LGTM!!

**Pull request merged**